rust_src/src/lists.rs

```diff
@@ -31,6 +31,12 @@ impl LispObject {
         }
     }
 
```

Comment:
This is correct but looks wrong. Is there a reason not to use `list.into()` instead of passing in the list?